code/_onclick/ai.dm

```diff
@@ -203,3 +203,8 @@
 
 /mob/living/silicon/ai/TurfAdjacent(turf/T)
 	return (GLOB.cameranet && GLOB.cameranet.checkTurfVis(T))
+
+/mob/living/silicon/ai/get_exp_list(minutes)
+	. = ..()
+
+	.[EXP_TYPE_JOB_AI] = minutes
```

Comment:
```suggestion
/mob/living/silicon/ai/get_exp_list(minutes)
	. = ..()
	var/static/datum/job/cyborg/ai_job = /datum/job/cyborg
	var/static/ai_job_name = initial(ai_job.name)
	.[ai_job_name] = minutes
```

Reply:
Handled through SSjob and grabbing a static reference to a job datum singleton we already have kicking around.